### ISSUE TITLE
Fix tile flooding in lighting system

### ISSUE THREAD

**Issue by Bravo555** [HopsonCommunity/Community-Game#208]

**Comment by Repertoi-e** [comment]
Can you give some details?

**Comment by Bravo555** [comment]
Yeah, sorry, should've done that. Basically i saw [this](https://github.com/HopsonCommunity/Community-Game/pull/207/files#diff-bfa9059d06734911c7d582531c0eb12dR31) and thought I'll make an issue so everyone can see. TODO comments in code aren't really visible unless you have some IDE plugin to highlight them.

**Comment by Repertoi-e** [comment]
TODO is about zombie targeting AI, not lightning

**Comment by Bravo555** [comment]
This comment appeared in PR which added lighting so I thought it must've something to do with the lighting. My bad.

**Comment by Bravo555** [comment]
And this TODO comment is probably a more like a little note to yourself than anything serious that we need to be aware of, right? In this case I probably shouldn't have done that and issue can be closed.

**Comment by Repertoi-e** [comment]
It's not really an issue.. Tile flooding for zombie AI would be nice but not a priority atm

**Comment by Bravo555** [comment]
Okay, so if this is a feature that we would like to add, the issue needs to stay open and be renamed accordingly.
Issues on github aren't really *issues*. It's more like kind of home for everything: todos, features, bugs, proposals, questions, etc. If something needs to be known for more than a one person it probably belongs here.
Thus, I encourage everyone to use the issues tab to submit bugs, planned features, your questions, and todos.

**Comment by Repertoi-e** [comment]
Yep.
Btw can you move the closed issues to the "Done" tab in the project window.
https://github.com/HopsonCommunity/Community-Game/projects/2

**Comment by Bravo555** [comment]
Nah, it's basically impossible on the mobile interface. I'll do it tomorrow.
Also, don't you have the possibility to do it? Because you should've. I'm also going to fix it tomorrow.

**Comment by Repertoi-e** [comment]
Either I don't or I don't know how.. Anyway I have to go now.. Don't worry about it
